src/cobra/summary/metabolite_summary.py

```diff
@@ -0,0 +1,399 @@
+"""Provide the metabolite summary class."""
+
+
+import logging
+from operator import attrgetter
+from textwrap import shorten
+from typing import TYPE_CHECKING, List, Optional, Union
+
+from pandas import DataFrame
+
+from cobra.flux_analysis import flux_variability_analysis, pfba
+from cobra.summary import Summary
+
+
+if TYPE_CHECKING:
+    from cobra.core import Metabolite, Model, Reaction, Solution
+
+
+logger = logging.getLogger(__name__)
+
+
+class MetaboliteSummary(Summary):
+    """
+    Define the metabolite summary.
+
+    Attributes
+    ----------
+    producing_flux : pandas.DataFrame
+        A table of only the producing fluxes.
```

Comment:
Can we say `A pandas DataFrame of only the producing fluxes.`?

Reply:
Can do. Although the explicit type is also there, that's why I the description to be a bit more natural language.

Reply:
I agree. Okay let's keep it the way it is then.

Reply:
Already changed :wink: